### ISSUE TITLE
How can I access to the provider from the session object?

### ISSUE THREAD

**Issue by prettydev** [nextauthjs/next-auth#500]
```
expires: "2020-08-28T07:29:27.766Z"
user:
email: "ylztguvzni_1596005541@tfbnw.net"
image: "https://scontent-sjc3-1.xx.fbcdn.net/v/t1.30497-1/cp0/c15.0.50.50a/p50x50/84628273_176159830277856_972693363922829312_n.jpg?_nc_cat=1&_nc_sid=12b3be&_nc_ohc=61UfcY8Xw30AX9mevIy&_nc_ht=scontent-sjc3-1.xx&oh=9356e3afe1a4e8fac5bb4e0c540c68e1&oe=5F45E4B8"
name: "Samantha Aledajbcjeiih Laverdetberg"
```
This is the session object after sign in with facebook account.
I want to extract the provider information from this object.
If impossible from this object, How can I get the provider infomation like google, facebook, and etc?

**Comment by iaincollins** [comment]
[There is a question about this in the FAQ](https://next-auth.js.org/faq#how-do-i-get-refresh-tokens-and-access-tokens-for-an-oauth-account).

tl;dr This one has come up a few times and there are a few existing issues relating to it.

It's probably going to be supported via dedicated API end points in the not too distant future, but right now you will need to grab them from the `accounts` table using the User ID (or use the `jwt` callback to persist them to the JWT on sign in, if using JSON Web Tokens).
